test/Altinn.Notifications.IntegrationTests/Utils/TestdataUtil.cs

```diff
@@ -9,6 +9,26 @@ namespace Altinn.Notifications.IntegrationTests.Utils;
 
 public static class TestdataUtil
 {
+    public static (NotificationOrder Order, SmsNotification Notification) GetOrderAndSmsNotification()
+    {
+        NotificationOrder order = NotificationOrder_SmsTemplate_OneRecipient();
+        order.Id = Guid.NewGuid();
+        var recipient = order.Recipients[0];
+        SmsAddressPoint? addressPoint = recipient.AddressInfo.Find(a => a.AddressType == AddressType.Sms) as SmsAddressPoint;
+
+        var smsNotification = new SmsNotification()
```

Comment:
How about we explicitly declare the type here? 🙂

Reply:
I see this is a test. May be it isn't needed to declare type explicitly here! 🤔

Reply:
I think the type is pretty clear here without declaring it explicitly and makes the code easier to read as the line is shorter. Makes more sense to avoid var if it is the output from a method with an unclear name.